Testscripts/Linux/docker_python_app.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+########################################################################
```

Comment:
indentation please

Reply:
> For CoreOS, it should support docker by default?
> For SUSE, we should add extra steps to support it? otherwise we need add skip.

Currently supported for RH/CentOS/Debian/Ubuntu. Test will be skipped for other distro